README.md

```diff
@@ -60,7 +62,7 @@ export DOMAIN_NAME=prombench.prometheus.io // Can be set to any other custom dom
 
 ---
 
-- Follow [Setting GitHub API and webhook](#setting-up-github-api-and-webhook-to-trigger-tests-from-comments)
+- Follow [Setting up GitHub webhook](#setting-up-github-webhook-to-trigger-tests-from-comments)
```

Comment:
we wouldn't need this anymore as the step would be dine when deploying prometheus-meta and grafana.

Reply:
not sure if I understood the change you wanted clearly, but I made some changes based on what i understood. please let me know if I need to make any other changes.